codice/ddf
# DDF-3705 Removed redundant pom properties declared in Codice-Parent

#### What does this PR do?
Added Codice-Parent as the parent project to DDF. Removed pom properties that are also defined in the Codice-Parent pom. Also adjusted poms to utilize property replacement where possible. Some minor pre-existing pom issues cleaned up as well.

#### Who is reviewing it? 
@garrettfreibott @austinsteffes @ahoffer @ricklarsen @shaundmorris @pklinef @clockard @djblue @lessarderic @coyotesqrl

#### How should this be tested?
Build, install and run to ensure compilation and runtime dependencies are satisfied. 

#### Any background context you want to provide?
This is part of an effort to move dependency management to a centralized location. We will need to evaluate dependencies that are using other versions that what is specified in the parent pom and justify or update them as needed. 

#### What are the relevant tickets?
[DDF-3705](https://codice.atlassian.net/browse/DDF-3705)
These changes are dependent on the following PR:
https://github.com/codice/codice-parent/pull/2

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
